### ISSUE TITLE
Moving default components like position, rotation and transform to different namespace

### ISSUE THREAD

**Issue by erdemserhatergen** [friflo/Friflo.Engine.ECS#14]
Having these components default from friflo results clash with rendering frameworks like fna and mono. I'm not a C# expert so couldn't find a good way to exclude those from my includes. 

Do you think it would be good idea to move them to different namespace for library?

**Comment by friflo** [comment]
Doing this will break compatibility.
So I don't want to do this at the moment.

**Comment by friflo** [comment]
Similar to https://github.com/friflo/Friflo.Engine.ECS/discussions/26

**Comment by popcron** [comment]
i agree with @erdemserhatergen and #26, but its already been put in place so yes technically it will break compatibility. but these types didnt need to be written as included in the first place! people can implement those themselves. maybe not for `EntityName` so easily as i think that should be a feature native to the lib without a component in use, but definitely for pos/rot/matrix components.

as for the namespace if theyre to be moved, `Friflo.ECS.Components` or `Friflo.ECS.BuiltIn` sound more fitting than `Math`.
